mongo-docker/start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Start mongo
+/tmp/entrypoint.sh mongod&
+
+sleep 5
+
+# Load data
+node /usr/local/app/data/reload_mongo.js
```

Comment:
So, will there be a time where the service is up, but not all data will be loaded yet?  Does it load fast enough that we won't care?